pkg/apis/pipeline/v1beta1/task_types.go

```diff
@@ -125,10 +125,11 @@ type Step struct {
 	//
 	// If Script is not empty, the Step cannot have an Command or Args.
 	Script string `json:"script,omitempty"`
+	// If step times out after Timeout, pod is terminated
```

Comment:
Let's generalize this comment to describe what the user will see, instead of how it will work.

```
    // Timeout is the time after which the step times out. Defaults to never.
    // Refer to Go's ParseDuration documentation for expected format: https://golang.org/pkg/time/#ParseDuration
```